client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -38,6 +41,10 @@ const mapStateToProps = (state) => ({
     listViewType: selectors.main.getCurrentListViewType(state),
     sortOrder: selectors.main.getCurrentSortOrder(state),
     sortField: selectors.main.getCurrentSortField(state),
+    users: selectors.general.users(state),
+    groups: selectors.main.itemGroups(state),
+    activefilter: selectors.main.activeFilter(state),
```

Comment:
`activefilter` and `currentSearch` is not required. Instead create a new selector for the search param you require (in this case, it would be a selector to get `coverage_user_id`)